### PR TITLE
Add details to see decrypted vault value

### PR DESCRIPTION
**Pull request by csillab**
1. Added workaround to display single encrypted value in ansible-vault. Thanks to @dirtyonekanobi from NTC.
2. Added some wording for clarity.

+label: docsite_pr

##### SUMMARY

Improving beginners guide for Network Engineers by adding an example how to display the original value of an encrypted single variable in a var file.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

Docs

##### ANSIBLE VERSION

```
ubuntu@ubuntu-xenial:~$ ansible --version
ansible 2.5.0rc2 (stable-2.5 2ee6a85463) last updated 2018/03/09 18:40:28 (GMT +000)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ubuntu/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ubuntu/ansible/lib/ansible
  executable location = /home/ubuntu/ansible/bin/ansible
  python version = 2.7.12 (default, Nov 20 2017, 18:23:56) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```
n/a
```

### CODE REVIEW

File: docs/docsite/rst/network/getting_started/first_inventory.rst [hunk 1]
```diff
@@ -243,6 +243,19 @@ Or with a prompt instead of the vault password file:
 
    ansible-playbook -i inventory --vault-id my_user@prompt first_playbook.yml
 
+To see the original value, you can use the debug module. Please note if your yaml file defines the `ansible_connection` variable (as we used in our example), it will take effect when you execute the command below. To prevent this please make a copy of the file without the ansible_connection variable for this. 
+
+.. code-block:: bash
```
Comment (gundalow) [inline docs/docsite/rst/network/getting_started/first_inventory.rst]: Think you need one of the console types here, which would fix the CI issue

**Comment by gundalow** [comment]
Wonder if this should go into the main Vault guide as well.

**Comment by csillab** [comment]
okay, pull request will follow... :)

**Review by dharmabumstead — APPROVED** [review]
Looks good to me.

**Review by acozine — APPROVED** [review]
Thanks for the PR. I appreciate your close reading of the guide!

**Pull request merged**